src/Symfony/Bridge/PhpUnit/composer.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "symfony/phpunit-bridge",
+    "type": "symfony-bridge",
+    "description": "Symfony PHPUnit Bridge",
+    "keywords": [],
+    "homepage": "http://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Nicolas Grekas",
```

Comment:
This is fine because he already had things like this merged https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/VarDumper/composer.json#L10 and it just makes sense. If the other components were attributed to @fabpot well that is because it was ok with their authors perhaps.